.github/workflows/auto-assign.yaml

```
@@ -5,6 +5,9 @@ on:
 
 jobs:
   assign:
+    permissions:
+      @ write permissions needed to assign the issue.
+      contents: write
```

Nice to see we can limit permissions to just jobs.

I wonder if the best-practice should have us putting default read permissions on the top-level in addition to the job-necessary write?

```suggestion
permissions:
  contents: read

jobs:
  assign:
    permissions:
      @ write permissions needed to assign the issue.
      contents: write
```

---

Thoughts here @obnoxxx ?

---

@BlaineEXE  wrote:

> Nice to see we can limit permissions to just jobs.

Thanks for confirming the approach!

> I wonder if the best-practice should have us putting default read permissions on the top-level in addition to the job-necessary write?

great idea, thanks! done.